README.md

```diff
@@ -86,15 +115,86 @@ IObservable<Unit> PushModal(IPageViewModel modal, string contract = null);
 IObservable<Unit> PushPage(IPageViewModel page, string contract = null, bool resetStack = false, bool animate = true);
 ```
 
-For example:
+### Example
 ```csharp
 OpenModal = ReactiveCommand
     .CreateFromObservable(() =>
         this.ViewStackService.PushModal(new FirstModalViewModel(ViewStackService)),
         outputScheduler: RxApp.MainThreadScheduler);
 ```
 
-For more examples, look inside the sample folder in the solution. 
+## Pass Parameters
+
+Version 2.0 added support for passing parameters when navigating.
+
+```csharp
+/// <summary>
+/// Pushes the <see cref="INavigable" /> onto the stack.
+/// </summary>
```

Comment:
No-where else in this code do we dump the source code in? Might be worth summarising the methods or something, or using a table like structure.

Reply:
Was just following the current structure of the README